business_requirement_deliverable_cost/report/br_deliverable_cost_report.py

```diff
@@ -0,0 +1,70 @@
+# -*- coding: utf-8 -*-
+# © 2016 Elico Corp (https://www.elico-corp.com).
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+from openerp import tools
+from openerp import fields, models
+
+
+class BusinessRequirementDeliverableCostReport(models.Model):
+    _name = "business.requirement.deliverable.cost.report"
+    _description = "Deliverable Cost Report"
+    _auto = False
+
+    name = fields.Char('Name', readonly=True)
+    description = fields.Char('Description', readonly=True)
+    partner_id = fields.Many2one('res.partner', 'Customer', readonly=True)
+    project_id = fields.Many2one('project.project', 'Master Project',
```

Comment:
@darshan-serpent Please review and change all fields parameters for models to layout indented as column, like this:
'project.project',
'Master Project',
readonly=True)